allure-report-face/src/components/testcase-table/TestcaseTableView.js

```diff
@@ -15,11 +16,13 @@ class TestcaseTableView extends DataGridView {
     initialize() {
         this.listenTo(settings, 'change:visibleStatuses', this.render);
         this.testCases = this.options.testCases.map((testcase, index) => Object.assign(testcase, {index: index + 1}));
+        this.statistics = countBy(this.testCases, testCase => testCase.status.toLowerCase());
```

Comment:
If you don't have a testcase with some status, button for that status will not be visible.

We need to initialize statistics with zero value for each known status.